polyn_elixir_client/lib/mix/tasks/polyn.gen.release.ex

```diff
@@ -0,0 +1,98 @@
+defmodule Mix.Tasks.Polyn.Gen.Release do
```

Comment:
I think this is a handy tool for people just starting out, but we I would be a bit cautious if people start asking for feature requests about the way we create/inject the `release.exs` file. I think we should just support this basic version here and then if they want anything more, we should just give people documentation that shows the basic version of the function they can add to whatever release process they are using.

Reply:
@mmmries That makes sense. I think that's a good boundary to set.